src/medCore/gui/toolboxes/medSelectorToolBox.cpp

```diff
@@ -35,7 +35,7 @@ medSelectorToolBox::medSelectorToolBox(QWidget *parent, QString tlbxId) :
     d->currentToolBox = NULL;
 
     d->chooseComboBox = new medComboBox;
-    d->chooseComboBox->addItem("Choose a toolbox");
+    d->chooseComboBox->addItem("* Choose a toolbox *");
```

Comment:
That's more of a hack than a solution. I would prefer you remove the first item before the sort, and put it back at the beginning afterwards.